### ISSUE TITLE
[REQUEST]: imputation from VCF

### ISSUE THREAD

**Issue by ClayBirkett** [maize-genetics/phg_v2#235]
### Description

In the previous version I could impute from VCF file
-ImputePipelinePlugin -imputeTarget pathToVCF

**Comment by yuru0503** [comment]
Imputations from VCF files or HapMap files are very useful and required for genomic selection purposes.

**Comment by Taechun** [comment]
Yeah, including the functions for imputing SNP genotypes from HapMap files is needed
